lib/screens/tutorial_screen.dart

```diff
@@ -81,9 +83,10 @@ class _TutorialScreenState extends State<TutorialScreen> {
                           decoration: const BoxDecoration(
                             image: DecorationImage(image: AssetImage(kTooltipBalloon), fit: BoxFit.contain),
                           ),
-                          child: Text(
+                          child: const AutoSizeText(
                             kWhyAppNeeded,
-                            style: TextStyle(fontSize: 12.sp, fontWeight: FontWeight.w400, color: EaselAppTheme.kWhite),
+                            maxFontSize: 18,
+                            style: TextStyle(fontSize: 18, fontWeight: FontWeight.w400, color: EaselAppTheme.kWhite),
```

Comment:
Should fontSize have .sp?

Reply:
No AutoSizeText will handle this automatically.